catalog/task/extract-index-image/0.2/README.md

```diff
@@ -0,0 +1,15 @@
+# publish-index-image
```

Comment:
@Troy876 this is my fault, not yours, but can you change this to `extract-index-image`?

Reply:
Should all good now